test/grape_test.rb

```diff
@@ -1,5 +1,7 @@
 require 'test_helper'
-require 'grape'
+TestHelper.silence_warnings do
+  require 'grape'
+end
```

Comment:
@rails-api/ams @rails-api/commit This commit is kind of unnecessary... might be easier to implement in grape.. :)